ext/ddtrace.stub.php

```diff
@@ -105,6 +105,11 @@ class SpanData {
          */
         public array $links = [];
 
+        /**
+         * @var string[] $peerServiceSources A sorted list of tag names used to set peer.service tag
```

Comment:
I think here's place for an extended documentation comment as to how peerServiceSources works in detail :-)

Reply:
@bwoebi  Good idea. Are you cool with https://github.com/DataDog/dd-trace-php/pull/2130/commits/fd8a9a3a2d8d54919ce694c5b90433e8fdd507fd